unlock-app/src/components/interface/RefundButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { ethers } from 'ethers'
+import withConfig from '../../utils/withConfig'
+
+interface RefundButtonProps {
+  accountAddress: string
+  lockAddress: string
+  config: {
+    providers: any
+    externalRefundContractAddress: string
+    env: 'dev' | 'test' | 'staging' | 'prod'
+  }
+}
+interface RefundButtonState {
+  refundInitiated: boolean
+}
+
+const lockAddressesByEnv = {
+  dev: '0x0AAF2059Cb2cE8Eeb1a0C60f4e0f2789214350a5',
+  test: '0x0AAF2059Cb2cE8Eeb1a0C60f4e0f2789214350a5',
+  staging: '',
+  prod: '0xB0ad425cA5792DD4C4Af9177c636e5b0e6c317BF',
```

Comment:
This, of course, is the address for the lock that sells EthWaterloo tickets on mainnet